spyder/widgets/mixins.py

```diff
@@ -971,14 +1017,24 @@ def is_space(move):
                 return not to_text_string(curs.selectedText()).strip()
 
             def is_special_character(move):

```

Comment:
Please add a docstring to this function. What characters is it supposed to catch?